src/test/java/racingcar/domainTest/RacingProcessTest.java

```diff
@@ -3,34 +3,32 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import camp.nextstep.edu.missionutils.test.NsTest;
 import java.util.ArrayList;
 import java.util.List;
 import org.junit.jupiter.api.Test;
-import racingcar.domain.RacingProcess;
 import racingcar.domain.Car;
+import racingcar.domain.RacingProcess;
 
 
 public class RacingProcessTest {
 
-    @Test
-    void 자동차목록을_받아서_진행도를_출력하는_기능_검사() {
-        RacingProcess race = new RacingProcess();
-        List<Car> carList = new ArrayList<>();
-        List<String> names = new ArrayList<>(List.of("pobi", "minju", "woni"));
-        List<Integer> progress = new ArrayList<>(List.of(1, 2, 0));
-        List<String> results;
-
-        for (int i = 0; i < 3; i++) {
-            carList.add(new Car(names.get(i), progress.get(i)));
-        }
-        results = new ArrayList<>(carList.size());
-        for (int i = 0; i < results.size(); i++) {
-            results.add(i, race.makingStatusBar(carList.get(i)));
-        }
-
-        assertThat(results.get(0)).contains("pobi : -");
-        assertThat(results.get(1)).contains("minju : --");
-        assertThat(results.get(2)).contains("woni : ");
+  @Test
+  void 자동차목록을_받아서_진행도를_출력하는_기능_검사() {
+    RacingProcess race = new RacingProcess();
+    List<Car> carLineUp = new ArrayList<>();
+    List<String> names = List.of("pobi", "minju", "woni");
+    List<Integer> progress = List.of(1, 2, 0);
+    List<String> results = new ArrayList<>();
+
+    for (int i = 0; i < 3; i++) {
+      carLineUp.add(new Car(names.get(i), progress.get(i)));
     }
+    for (int i = 0; i < carLineUp.size(); i++) {
+      results.add(race.makingStatusBar(carLineUp.get(i)));
+    }
+
+    assertThat(results.get(0)).contains("pobi : -");
+    assertThat(results.get(1)).contains("minju : --");
+    assertThat(results.get(2)).contains("woni : ");
```

Comment:
해당 문자열이 완전히 동일한 문자열인지를 테스트하고 싶다면 `isEqualTo()` 메서드를 사용해볼 수 있어요!

```suggestion
    assertThat(results.get(0)).isEqualTo("pobi : -");
    assertThat(results.get(1)).isEqualTo("minju : --");
    assertThat(results.get(2)).isEqualTo("woni : ");
```